Gopkg.lock

```diff
@@ -1331,6 +1331,14 @@
   revision = "c6a59be0ce122566695fbd5e48a77f8f10c8a63a"
   version = "v1.0.0"
 
+[[projects]]
```

Comment:
Doesn't this require an update to `Gopkg.toml`?

Reply:
Thanks for spotting that 🙏 I don't know how the `Gopkg.toml` was left out 🤔